app/com/arpnetworking/kairos/client/models/MetricsQuery.java

```diff
@@ -116,7 +116,7 @@ public Builder setStartTime(final DateTime value) {
          * @param value the end time
          * @return this {@link Builder}
          */
-        public Builder setEndTime(final DateTime value) {
+        public Builder setEndTime(final Instant value) {
```

Comment:
Not your fault; can you mark the argument `@Nullable`?

Reply:
done